acdc-ws/app/services/MetricReporter.scala

```diff
@@ -0,0 +1,111 @@
+package services
+
+import com.typesafe.config.ConfigFactory
+import io.prometheus.client._
+import io.prometheus.client.exporter.common.TextFormat
+import io.prometheus.client.hotspot.DefaultExports
+import play.api.Configuration
+
+import java.io.StringWriter
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+object Metrics {
+
+  val DefaultBucketValues: Array[Double] = Array(
+    .005,
+    .010,
+    .025,
+    .050,
+    .100,
+    .250,
+    .500,
+    1.0,
+    2.5,
+    5.0,
+    10.0,
+    15.0,
+    60.0,
+    300.0,
+    1200.0,
+    Double.PositiveInfinity
+  )
+
+  val httpDurationSeconds: Histogram = Histogram.build
```

Comment:
let's put those in the injected metric service instead of `object` here.

Reply:
Sounds good.

Reply:
I moved these Prometheus objects into the Prometheus class, which implements the Metric trait in the latest refactor.